music21/serial.py

```diff
@@ -1048,42 +1048,58 @@ def areCombinatorial(self, transType1, index1, transType2, index2, convention):
         :meth:`~music21.serial.zeroCenteredTransformation` and
         :meth:`~music21.serial.originalCenteredTransformation` explanations of these conventions.
 
+        First, let's take a row we know to have a combinatoriality pair:
+
         >>> moses = serial.getHistoricalRowByName('SchoenbergMosesAron')
         >>> moses.pitchClasses()
         [9, 10, 4, 2, 3, 1, 7, 5, 6, 8, 11, 0]
-        >>> moses.areCombinatorial('P', 1, 'I', 4, 'zero')
+
+        Combinatoriality holds here between P0 and I3
+        >>> moses.areCombinatorial('P', 0, 'I', 3, 'zero')
+        True
+
+        By definition what's true for :meth:`~music21.serial.zeroCenteredTransformation`
+        is also true for :meth:`~music21.serial.originalCenteredTransformation`:
```

Comment:
Pre-existing bug, but these links don't render, because these are methods on `ToneRow`. Adding `.ToneRow.` fixes it -- are you game to replace the two sets here and at the top of the docstring, and the third (set) of links on the `findZero...` and `findOriginal...` methods?

Reply:
Sure thing - thanks for catching this.